src/hooks/useRedirectLoggedInEffect.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import { useIsLoggedIn } from './useIsLoggedIn';
+
+export const useRedirectLoggedInEffect = () => {
+  const navigate = useNavigate();
+  const isLoggedIn = useIsLoggedIn();
+
+  useEffect(() => {
+    if (!isLoggedIn) return;
+    navigate('/');
+  }, [isLoggedIn, navigate]);
```

Comment:
if 조건에는 부정문보다 긍정문을 쓰는게 읽기 편하다고 생각하는데 이렇게 바꾸면 어떨까요

```suggestion
    if (isLoggedIn) {
      navigate('/');
    }
```

Reply:
바로 청소 하겠습니다